Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracks_cff.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+ALCARECOPPSCalMaxTracksFilter = hlt.hltHighLevel.clone(
+  TriggerResultsTag = cms.InputTag("TriggerResults","","HLTX"),
```

Comment:
I think this line could be dropped altogether.

Reply:
It is added for the purpose of testing, as our fake ALCARAW data are generated with `HLTX` process.
I will think of a way to move it to the testing script.

Reply:
Done in https://github.com/cms-sw/cmssw/pull/36273/commits/18c83a8f88a9ba2168b0438515eb0d211ecdd53c